client/my-sites/checkout/checkout-thank-you/domains/thank-you-content/domain-registration.tsx

```diff
@@ -24,33 +25,58 @@ const DomainRegistrationThankYouProps = ( {
 			selectedSiteSlug,
 		},
 		'REGISTRATION',
-		false
+		true
 	);
 
+	const createSiteStep = {
+		stepKey: 'domain_registration_whats_next_create-site',
+		stepTitle: translate( 'Add a site' ),
+		stepDescription: translate( 'Choose a theme, customize and launch your site.' ),
+		stepCta: (
+			<FullWidthButton
+				href={ createSiteFromDomainOnly( domain, null ) }
+				busy={ false }
+				disabled={ false }
+			>
+				{ translate( 'Create a site' ) }
+			</FullWidthButton>
+		),
+	};
+
+	const viewDomainsStep = {
+		stepKey: 'domain_registration_whats_next_view_domains',
+		stepTitle: selectedSiteSlug
+			? translate( 'Organize your domains' )
+			: translate( 'Manage your domains' ),
+		stepDescription: selectedSiteSlug
+			? translate(
+					'Set up a primary domain, connect other domains and make sure people can find your site.'
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( 'Set up a primary domain, connect other domains and make sure people can find your site' )` *ES Score: 10*